app/style/content/preferences/account.less

```diff
@@ -107,6 +113,13 @@
   }
 }
 
+.preferences-history-backup-import-field {
+  > input {
+    position: absolute;
+    left: -9999px;
```

Comment:
@AndyLnd Do we have `-9999px` somewhere else in the code so that we can reference that variable?